randovania/layout/description_migration.py

```diff
@@ -695,6 +695,16 @@ def fix_elevator(conn: dict[str, str], old_name: str, new_name: str) -> None:
             )
 
 
+def _migrate_v35(data: dict) -> None:
+    game_modifications = data["game_modifications"]
+
+    for index, game in enumerate(game_modifications):
+        for region in game["locations"].keys():
+            for pickup in game["locations"][region].keys():
+                if "Energy Transfer Module" in game["locations"][region][pickup]:
```

Comment:
I would at least comment why it's doing this - I also don't know if this can potentially break anything? @henriquegemignani might know better why some games had `Energy Transfer Module for Player 2` and the effects of not keeping that.

Reply:
Ideally they'd all be in that format, but I'm gonna guess bad old migration?

Reply:
@SchwartzGandhi can you add the comment please?